db/migrations/postgres/000097_create_posts_priority.up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS postspriority (
+    postid VARCHAR(26) PRIMARY KEY,
+    channelid VARCHAR(26) NOT NULL,
+    priority VARCHAR(32) NOT NULL,
```

Comment:
Why 32?

Reply:
I think there is a plan to support custom labels in the future. I thought 32 would be a good number, but it is indeed random.